huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_customer_gateway_test.go

```diff
@@ -90,7 +90,7 @@ func testCustomerGateway_basic(name string) string {
 	return fmt.Sprintf(`
 resource "huaweicloud_vpn_customer_gateway" "test" {
   name = "%s"
-  ip   = "192.168.1.1"
+  ip   = "172.16.1.1"
```

Comment:
What's the different?
Is it cause of the peer parameter?
```
peer_subnets        = ["172.16.55.0/24"]
```

Reply:
just make it different from the vpn_gateway to avoid error